net-misc/openconnect/files/openconnect-7.07-libressl.patch

```diff
@@ -0,0 +1,58 @@
+diff --git openssl-esp.c openssl-esp.c
```

Comment:
In that case, you could manually add a description to your own patch, or we can just wait until the next openconnect release.